### ISSUE TITLE
Recurring Events & RRULE - Displaying Time when allDay is true

### ISSUE THREAD

**Issue by nbcnc** [fullcalendar/fullcalendar#4576]
### FullCalendar V4

Issue when displaying recurring events using the RRULE library.
When `allDay` is set to `true` the time of the event is still displayed.

https://jsfiddle.net/marcparillo/79mk20p8/

```
 var calendarEl = document.getElementById('calendar');
 var calendar = new FullCalendar.Calendar(calendarEl, {
   plugins: ['dayGrid', 'rrule'],
   events: [{
     id: 1,
     allDay: true,
     title: 'my recurring event',
     rrule: 'DTSTART:20190301T073000Z\nRRULE:FREQ=WEEKLY;INTERVAL=1;BYDAY=MO'
   }]
 });
 calendar.render();
```
<img width="165" alt="Screen Shot 2019-03-22 at 11 10 40 AM" src="https://user-images.githubusercontent.com/27741514/54832769-2e2b3d80-4c93-11e9-8da2-2fa52844d865.png">

I expected the calendar would not show the time when `allDay` is true.
I'm currently using this work-around to remove the time myself.

```
eventRender:                info => {
     const isAllDay = info.event.extendedProps.allDay;
     const isRecurring = info.event.extendedProps.recurring == 1;
     if (isAllDay && isRecurring) {
        const time = info.el.querySelector('.fc-time');
        time && time.remove();
     }
}
```

**Comment by acerix** [comment]
Confirmed, thanks for the detailed report and workaround.

**Comment by arshaw** [comment]
fixed in [v4.0.2](https://github.com/fullcalendar/fullcalendar/releases/tag/v4.0.2)!
